javascript-forms/form-values.js

```diff
@@ -0,0 +1,14 @@
+const $contactForm = document.querySelector('#contact-form');
+const formValues = {};
+
+function completedForm(event) {
+  formValues.name = $contactForm['user-name'].value;
+  formValues.email = $contactForm['user-email'].value;
+  formValues.message = $contactForm['user-message'].value;
+
+  console.log(formValues);
+  event.preventDefault();
```

Comment:
This should be at the top of the function, because it's the first thing we want to run